week02-03/homework_1/README.md

```diff
@@ -171,13 +171,13 @@ print 4.0 * n_hits / float(n_trials), math.sqrt(var / n_trials)
 
 The program of Section C1 used the exact value of pi in order to compute the error. In a real calculation, we don't know this result beforehand. One therefore replaces pi through its best estimate, the mean value <Obs>. Using
 
-<(Obs−pi)^2>∼<Obs^2>−pi^2∼<Obs^2>−<Obs>^2,
+[(Obs−pi)^2]∼[Obs^2]−pi^2∼[Obs^2]−[Obs]^2,
```

Comment:
This notation is not equivalent. The <.> operator is a convenient notation for the mean value. By substituting E(X+Y) with <E+Y> we make less visual noise.

Reply:
I understand that You wish to follow suitable conventions. 

Unfortunately **Markdown**, which is the formatting language for **GitHub** is not really compatible with this notation. If you look closely at the description of the commit I made, I explain that **'<'** and **'>'** are **not parsed correctly**. What you get is that **the formulas don't even show up properly**! So, yes, the problem of visual noise is important, but I was addressing the issue that the formulas don't even show up in the browser.

I thought that, since this is **just a notational convention**, we could use instead of **'<.>'** the symbols **'['** and **']'**.  I understand that it would be better if we kept Your convention, so that is why I am opening a **new pull request**, where I have fixed the problem with visualizing the formulas, and kept to the convention, but please keep in mind that **the workaround is a bit rough** because I use the **character entity references** `&lt;` and `&gt;` in place for the symbols `<` and `>`.